### DISCUSSION TITLE
Broken 'Blog' links in the footer of the official website

### DISCUSSION THREAD

**Discussion by lucydodo** [getredash/redash#6849] [General]
Redash team, Hello
I have noticed that the link to 'Blog' in the footer navigation of your official homepage is brokend and I would like to inform about it.

![image](https://github.com/getredash/redash/assets/64626662/8ffc3c6f-4829-4b0e-81bc-179d28067e9e)
You can find above footer navigation in `https://redash.io`
Broken link: `https://blog.redash.io`

> I thought about opening an issue, but realized it's not a problem with the software itself, so I'm leaving it in Discussion.

Thank you. :)

**Comment by justinclift** [comment]
Thanks. :smile:

Any interest in making a PR to remove that old Blog entry from the footer?

The website source repo is here, if that's useful: https://github.com/getredash/website :smile:

**Comment by lucydodo** [comment]
Cool, I'll make a PR. :)

**Comment by justinclift** [comment]
Merged your PR (thanks heaps!), so I'll close this. :smile:

**Reply by lucydodo** [reply]
Thanks :)
